aptos-move/e2e-benchmark/data/calibration_values.tsv

```diff
@@ -12,14 +12,14 @@ VectorPicture { length: 30720 }	34	0.924	1.081	6900.0
 VectorPictureRead { length: 30720 }	34	0.938	1.089	6900.0
 SmartTablePicture { length: 30720, num_points_per_txn: 200 }	34	0.972	1.074	42970.1
 SmartTablePicture { length: 1048576, num_points_per_txn: 300 }	34	0.960	1.066	73865.4
-ResourceGroupsSenderWriteTag { string_length: 1024 }	34	0.898	1.062	16.2
+ResourceGroupsSenderWriteTag { string_length: 1024 }	34	0.898	1.062	19.2
```

Comment:
This PR does not touch anything in the execution stack, so this most be from previous PRs. (Perhaps permissioned signers? @igor-aptos

Reply:
Seems like signer has indeed added noise - here it the recalibration PR - #15778

generated by doing:
```
./testsuite/single_node_performance_calibration.py --time-interval=3d
./testsuite/single_node_performance_calibration.py --move-e2e --time-interval=3d
```

note - this particular line didn't regress as much as you saw from the first run - you can see your second run much closer to the original value. But I assume all the changes create way too much noise anyways.